vmc/resource_vmc_sddc.go

```diff
@@ -364,8 +367,47 @@ func resourceSddcUpdate(d *schema.ResourceData, m interface{}) error {
 	sddcID := d.Id()
 	orgID := (m.(*ConnectorWrapper)).OrgID
 
+	// Convert sddc from 1node to default
+	toConvertSddc := false
+	if d.HasChange("sddc_type") {
+		oldTmp, newTmp := d.GetChange("sddc_type")
+		oldType := oldTmp.(string)
+		newType := newTmp.(string)
+
+		// Validate for convert type params
+		if oldType == "1NODE" && (newType == "" || newType == "DEFAULT") {
+			oldTmp, newTmp := d.GetChange("num_host")
+			oldNum := oldTmp.(int)
+			newNum := newTmp.(int)
+			if oldNum != 1 || newNum != 3 {
+				return fmt.Errorf("Scale SDDC must from 1 host to 3 hosts")
```

Comment:
Change error message to something like "Scale up for SDDC must be from 1 host to 3 hosts"